src/cobalt_model/datetime.rs

```diff
@@ -29,6 +29,10 @@ impl DateTime {
         self.0.format("%Y-%m-%d %H:%M:%S %z").to_string()
     }
 
+    pub fn prefix(&self) -> String {
```

Comment:
This seems too specialized to go on this struct

Reply:
But I can't access the inside chrono date, shall I make an accessor to it?

Reply:
Options
1. Create a `Date` object, create a getter on `DateTime` for it, have a `Date::format`
2. Have a `format_date`
3. Use `impl ops::Deref for DateTime ` to access Chrono
4. Use `impl convert::From<DateTime> for chrono::DateTime<chrono::FixedOffset> ` to create a Chrono
5. Do our own formatting, taking advantage of ` impl chrono::Datelike for DateTime`

Reply:
Do you have a preference? I have a slight pref for solution 2, doing a `format_date`, and refactor current `format` to call this new `format_date`.

Reply:
`format` is doing its own formatting and shouldn't be refactored with `format_date`.

When sharing code, the thing I like to point out is if we are sharing requirements or implementation.  If it is just implementation, we shouldn't reduce "duplication".  In this case, there are no shared requirements (nothing dictates that `format`s start match our filename start).

Similarly, this policy for filenames is exclusive to filenames and might not have shared requirements with other code that might want dates.  For this reason, I don't recommend centralizing it on `DateTime`.

I'm leaning towards (3) as the simplest option.

Reply:
about refactoring, I was thinking of something like that:
```rust
pub fn format(&self) -> String {
        self.format_date("%Y-%m-%d %H:%M:%S %z")
    }

    pub fn format_date(&self, format: &str) -> String {
        self.0.format(format).to_string()
    }
```

but I see the point of not sharing code in this case, I tends to factorise as much as possible, but I can see the issue of bloated classes if followed to strictly